src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -264,7 +275,25 @@
         limit-insufficient?                         (> (controlled-input/numeric-value input-state)
                                                        current-limit)
         should-try-again?                           (and (not limit-insufficient?) no-routes-found?)
-        current-address                             (rf/sub [:wallet/current-viewing-account-address])]
+        current-address                             (rf/sub [:wallet/current-viewing-account-address])
+        owned-eth-token                             (rf/sub [:wallet/token-by-symbol
+                                                             (string/upper-case
+                                                              constants/mainnet-short-name)
+                                                             enabled-from-chain-ids])
+        not-enough-asset?                           (and
+                                                     (or no-routes-found? limit-insufficient?)
+                                                     (not-empty sender-network-values)
+                                                     (if (= token-symbol
+                                                            (string/upper-case
+                                                             constants/mainnet-short-name))
```

Comment:
Isn't it better to do this the other way around? Instead of transforming the constant to uppercase, transform the `token-symbol` to lowercase, and then compare? While it's unlikely, what if the `token-symbol` is `Eth`? We can be certain that the constant is `eth` (in all lowercase), but can we be as sure about the` token-symbol`?